src/sources/tripadvisor/pois/convert.rs

```diff
@@ -157,6 +160,11 @@ pub fn build_poi(property: Property, geofinder: &AdminGeoFinder) -> Result<(u32,
     ]
     .into_iter()
     .filter_map(|(key, val)| Some((key.to_string(), val?)))
+    .chain(
+        (names.0)
```

Comment:
these lines add all names in different languages into properties?

Reply:
Yes, these name properties typically are in the form `{ key: "fr", value: "Tour Eiffel" }`.

If you ask it may suggest that this needs more comments to explain what's going on ? :sweat_smile:

Reply:
I was surprised by `names.0`, meaning, we only take the first language ?

Reply:
It comes from [`I18nProperties`](https://github.com/CanalTP/mimirsbrunn/blob/master/libs/places/src/i18n_properties.rs#L10) being defined as [tuple struct](https://doc.rust-lang.org/book/ch05-01-defining-structs.html#using-tuple-structs-without-named-fields-to-create-different-types) of one field in mimirsbrunn.

I agree this is not extremely readable, a solution would be to implement `Deref` and `DerefMut` in mimirsbrunn to allow calling `Vec`'s methods on `I18nProperties` directly, which would be more obvious.